dbflute-engine/src/main/java/org/dbflute/helper/jdbc/sqlfile/DfSqlFileRunnerBase.java

```diff
@@ -421,6 +424,10 @@ protected void closeReader(Reader reader) {
                 }
 
                 if (sql.trim().endsWith(_runInfo.getDelimiter())) {
+                    // Block Comment
+                    if (DfStringUtil.count(sql, "/*") != DfStringUtil.count(sql, "*/")) {
```

Comment:
Give me a time to analyze this statement.
This class is so complex and it needs to test at local environment.